GithubRepository/test/repositoryUtils.test.js

```diff
@@ -0,0 +1,107 @@
+const { filterRepositoriesWithMoreThanFiveStars } = require('../src/repositoryUtils.js');
+
+
+describe('filterRepositoriesWithMoreThanFiveStars', () => {
+    test('should return repositories with more than 5 stars', () => {
```

Comment:
Let's avoid using `should` read this [little article](http://bretthard.in/post/tests-should-not-begin-with-should)